### ISSUE TITLE
hardhat-zksync-upgradeable dependency issue with @openzeppelin/upgrades-core

### ISSUE THREAD

**Issue by statskiy-sovetnik** [matter-labs/hardhat-zksync#361]
Running `yarn hardhat verify <myAddress> --network zcSyncTestnet` results in the following error (it seems like the upgrades-core package doesn't have the ast-dereferencer module now):
  
![image](https://github.com/matter-labs/hardhat-zksync/assets/32808977/636216d4-3083-4bd4-bc48-cdfd705bf645)

The plugins seem to be imported in correct order in my hardhat.config.ts; hardhat-zksync-verify plugin is installed with version 0.1.8

![image](https://github.com/matter-labs/hardhat-zksync/assets/32808977/1531816c-093b-495e-aec8-b6200083f5e5)

**Comment by statskiy-sovetnik** [comment]
Manually installing the previous version @openzeppelin/upgrades-core worked for me

**Comment by anhlh-edsolabs** [comment]
> Manually installing the previous version @openzeppelin/upgrades-core worked for me

Which previous version of `@openzeppelin/upgrades-core` did you install to make it work?

**Comment by statskiy-sovetnik** [comment]
> > Manually installing the previous version @openzeppelin/upgrades-core worked for me
> 
> Which previous version of `@openzeppelin/upgrades-core` did you install to make it work?

1.27.3

**Comment by FilipTxFusion** [comment]
Hi @statskiy-sovetnik @anhlh-edsolabs, this issue was caused by hardhat-zksync-upgradable plugin not yet supporting ethers v6 upgrade, while the latest openzeppelin packages have new major versions that include this change.

We are actively working on this ethers upgrade, but in the meantime we released a new version of our upgradable plugin - 0.1.3 - that fixes this dependency issue.

Thanks for pointing it out!
